db/migrate/20161128193435_add_researcher_quote_to_project_contents.rb

```diff
@@ -0,0 +1,7 @@
+class AddResearcherQuoteToProjectContents < ActiveRecord::Migration
+  def change
+    add_column :project_contents, :researcher_quote, :text
+
+    add_index :project_contents, :researcher_quote
```

Comment:
we won't need this unless we actually query on this directly. indexes take up resources on the database so we should only add them if we have an actual query path that uses them.